cargo-shuttle/src/lib.rs

```diff
@@ -952,7 +973,29 @@ impl Shuttle {
                 _ => CommandOutcome::Ok,
             })
         } else {
-            println!("Deployment has not entered the running state");
+            println!("{}", "Deployment has not entered the running state".red());
+            println!();
+            match last_state {
+                Some(shuttle_common::deployment::State::Stopped) => {
+                    println!("State: Stopped - Deployment was running, but has been stopped by the user.")
+                }
+                Some(shuttle_common::deployment::State::Completed) => {
+                    println!("State: Completed - Deployment was running, but stopped running all by itself.")
+                }
+                Some(shuttle_common::deployment::State::Unknown) => {
+                    println!("State: Unknown - This may be because deployment was in an unknown stateWe never expect this state and entering this state should be considered a bug.")
```

Comment:
```suggestion
                    println!("State: Unknown - This may be because deployment was in an unknown state. We never expect this state and entering this state should be considered a bug.")
```